hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapred/MapTask.java

```diff
@@ -930,7 +930,7 @@ private long getOutputBytes(List<Statistics> stats) {
     // spill accounting
     private int maxRec;
     private int softLimit;
-    boolean spillInProgress;;
+    private boolean spillInProgress;
```

Comment:
no need to add `private` here as part of this change

Reply:
Thanks for your suggestion, I will fix it.